lib/elixir/test/elixir/application_test.exs

```diff
@@ -21,6 +21,20 @@ defmodule ApplicationTest do
     assert Application.get_env(:elixir, :unknown, :default) == :default
   end
 
+  test "application information" do
+    started = Application.started_applications
+    assert started |> is_list
```

Comment:
But I just love pipes :)

If that's not the code style here then happy to change though.